frontend/src/providers/AppProviders.js

```diff
@@ -3,12 +3,13 @@ import { createContext, useState } from 'react';
 import { ThemeProvider } from 'styled-components';
 import { theme } from 'assets/styles/theme';
 import { GlobalStyle } from 'assets/styles/GlobalStyles';
+import avatar1 from 'mocks/images/avatars/sample-avatar1.jpg';
```

Comment:
zamiast `avatar1` dałbym nazwę bardziej pasującą, np `defaultUserAvatar `

Reply:
Dodatkowo jeśli to domyślny obrazek to musiałby być gdzieś indziej niż w mockach. Folder mock powinien służć nam tylko do testów.
Może` /frontend/src/assets/images/` ?

Reply:
Jak coś to tak. Musiałbyś tam umieścić kopię tego obrazka :))) Jeden do testów, a drugi jako domyślny obrazek.

Reply:
Jak coś to nie jest defaultowy obrazek, tylko avatar tego użytkownika, którego zrobiliśmy na sztywno.

Reply:
Myślę, że na to powinien być nowy task. Bo ja tylko wyrzuciłem commentForma ;)